src/main/java/techcourse/myblog/domain/Article.java

```diff
@@ -1,4 +1,81 @@
 package techcourse.myblog.domain;
 
+import java.util.Objects;
+
 public class Article {
+    private static final int EMPTY_ID = -1;
+
+    private int id;
+    private String title;
+    private String coverUrl;
+    private String contents;
+
+    public Article(int id, String title, String coverUrl, String contents) {
+        this.id = id;
+        this.title = title;
+        this.coverUrl = coverUrl;
+        this.contents = contents;
+    }
+
+    public static Article of(String title, String coverUrl, String contents) {
+        return new Article(EMPTY_ID, title, coverUrl, contents);
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요? setter 메서드는 도메인의 핵심 개념이나 의도를 코드에서 사라지게 해요.

```suggestion
    public void changeTitle(String title) {
```